### PR TITLE
feat: show double confirmation when ignoring asset

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/composables/blockchain/use-account-addition-notifications.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export function useAccountAdditionNotifications(): UseAccountAdditionNotificatio
         address: account.address,
         list: listOfFailureText.join('\n'),
       }),
-      severity: Severity.INFO,
+      severity: Severity.WARNING,
```
Comment (kelsos) [inline frontend/app/src/composables/blockchain/use-account-addition-notifications.ts:85]: I am thinking that the severity in this actually depends on the data.

We probably need to identify the different cases and change the severity based on the status.

File: frontend/app/src/components/asset-manager/managed/ManagedAssetTable.vue [hunk 2]
```diff
@@ -136,15 +137,28 @@ function getAsset(item: SupportedAsset) {
...
+    show({
+      message: t('ignore.confirm.message', {
+        asset: symbol || name,
+      }),
+      title: t('ignore.confirm.title'),
+      type: 'warning',
+    }, async () => {
+      await ignoreAsset(identifier);
+    });
+  }
```
Comment (kelsos) [inline frontend/app/src/components/asset-manager/managed/ManagedAssetTable.vue:161]: Would it make sense to move this to the `useIgnoreAssetStore` and reuse it to the other component too?

File: frontend/app/src/composables/blockchain/use-account-addition-notifications.ts (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/composables/blockchain/use-account-addition-notifications.ts:85]: okay gonna revert this and do this in different PR then

**Review by cleanerzkp — APPROVED** [review]
LGTM

**Pull request merged**